package.json

```diff
@@ -21,7 +21,8 @@
     "release-it": "*"
   },
   "dependencies": {
-    "@eeacms/volto-blocks-form": "^1.0.0"
+    "@eeacms/volto-blocks-form": "^1.0.0",
+    "volto-slate": "^1.0.0"
```

Comment:
@nileshgulia1 :smile: No, the accordion block should work also with DraftJS

Reply:
I added this because the tests are failing on CI, but passing on local. As in CI the generated volto app only supports `draftjs` and do not have `volto-slate` as an addon.

Reply:
tried adding conditional, let's see if it works.